.gitignore

```diff
@@ -41,3 +41,5 @@ verilator-config-version.cmake
 **/_build/*
 **/obj_dir/*
 /.vscode/
+/.idea/
+/cmake-build-*/
```

Comment:
These are the project path and default CMake build output path for CLion, a popular C/C++ IDE